src/sentry/sentry_metrics/client/base.py

```diff
@@ -64,3 +61,10 @@ def distribution(
         has been registered in the UseCaseID enum.
         """
         raise NotImplementedError()
+
+    def close(self) -> None:
+        """
+        Calling this is required once we are done emitting metrics
```

Comment:
This states it is required, but we are not actually calling it anywhere.

Reply:
I believe we use this in tests now, but if we don't we might want to remove it